pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py

```diff
@@ -197,38 +164,38 @@ def get_supported_device_raid_levels(device_type):
     If this changes, then so should this function, but at this time it
     is not clear what RAID levels should be offered for other device types.
 
+    :param device_tree: a proxy of a device tree
     :param int device_type: one of an enumeration of device types
     :return: a set of supported raid levels
-    :rtype: a set of instances of blivet.devicelibs.raid.RAIDLevel
+    :rtype: a set of strings
     """
     if device_type == DEVICE_TYPE_MD:
-        supported = set(raid.RAIDLevels(["raid0", "raid1", "raid4", "raid5", "raid6", "raid10"]))
-    else:
-        supported = set()
+        supported = {"raid0", "raid1", "raid4", "raid5", "raid6", "raid10"}
+        levels = set(device_tree.GetSupportedRaidLevels(DEVICE_TYPE_MD))
+        return levels.intersection(supported)
 
-    return get_supported_raid_levels(device_type).intersection(supported)
+    return set()
 
 
-@memoizer
```

Comment:
Why?

Reply:
These functions are not called so often as before, because the caller remembers the supported raid levels now. Therefore, it is not necessary to cache the results.